src/pages/ProfilePage.js

```diff
@@ -200,15 +201,16 @@ function ProfilePage(props) {
                             ) : null}
                             {shouldShowLocalTime && <AutoUpdateTime timezone={timezone} />}
                         </View>
-                        {!isCurrentUser && Boolean(login) && (
-                            <MenuItem
-                                title={`${props.translate('common.message')}${displayName}`}
-                                icon={Expensicons.ChatBubble}
-                                onPress={() => Report.navigateToAndOpenReport([login])}
-                                wrapperStyle={styles.breakAll}
-                                shouldShowRightIcon
-                            />
-                        )}
+                        {!isCurrentUser &&
+                            !Session.isAnonymousUser(
```

Comment:
```suggestion
                            !Session.isAnonymousUser() && (
```

Reply:
Wow, not sure what happened there. Thanks for catching.